### DISCUSSION TITLE
Photos are all doubled up

### DISCUSSION THREAD

**Discussion by maxdd** [immich-app/immich#9201] [General]
Hello,
in my android immich app, is it normal to have double the photos e.g. those on the server and those on the phone?
I'm not sure whether this is right.
I did a backup from my phone to the server

I would expect to have a photo which is flagged as both local and remote rather than a copy of it

Regards,

**Comment by LMSSonos** [comment]
@maxdd did you find a solution? I had it working but since the update to version 1.118 I see all photos on android twice (once with synced and once without the synced icon)

**Reply by maxdd** [reply]
I believe it turned out to be some strange bug or situation, after I've redeployed the docker and resynced all the photos it then started to work normally

**Reply by LMSSonos** [reply]
I think I had 2 issues 
1) Migrated to a new android phone, saw that it was still running as the old device within immich UI (Devices)
2) I still copy files from the phone (behavior before using immich to backup) to a local library which is used within immich, now the files are duplicate on the server, in the web-ui, I see them only once, in the android app double, maybe it has something to do with immich moving to new APIs, but not sure

**Reply by maxdd** [reply]
Would running the server side job that check for doubles be helpful?

**Reply by LMSSonos** [reply]
It shows me about 2000 pictures as duplicates/similar, that is exact the problem, that I have them double. I need to change my workflow for the future - I don't think there is anything in immich to change, because just to explain what I do

1) I backup to immich
2) I have a readonly library, regularly, I copy all pictures from my phone to the PC, delete the ones I don't want to keep, then copy them to the readonly library. 

This is what creates the duplicate, and to be correct, it also shows doubles on the web. I don't want to run the deduplication to delete all pictures, as it also deletes similar ones. I also don't know what happens because immich will no be able to delete the readonly files. 

So I cleanup the folders by myself when processing the pictures

**Reply by LMSSonos** [reply]
> I believe it turned out to be some strange bug or situation, after I've redeployed the docker and resynced all the photos it then started to work normally

@maxdd - sorry I investigated further on that issue and even if I take a new photo, it will be backed up and then shows twice in immich on android. Question, what did you do when you wrote `resynced`?

I think it is android related, as the issue is only in the mobile app and not on the web

**Reply by maxdd** [reply]
Can you tell me what's your backup conf in the app? Which folder have you selected? In my case I had to delete everything server side are re-launch the backup.
This is because in the first iteration something I ignored occured and immich was synching with the server but was not able to recognize that my Camera/DCIM folder contained the same images so it would always display them twice (1 local and 1 which was of a previous backup)

**Reply by LMSSonos** [reply]
Yes, in android I have selected the `Camera` folder. 

I don't want to delete everything on the server as I will loose all the face assignments I did (was a lot of work)

Obviously the issue does not happen on the server. I will try later to see if I get any output from the logs using `adb`

**Reply by maxdd** [reply]
The issue is in your app because your Camera folder and server folder are not recognized the same in terms of their content.
My suggestion is to create a parallel deployment or simply change (temporarily) the server folder in which you save the synced photos to see whether that resolve the issue

**Reply by LMSSonos** [reply]
thanks, will do this. I have switched to a new android phone and maybe this could be one of the main reasons. I will first try to remove everything on android and make a fresh install. I will also logout the device from the web UI, I will keep you posted

**Reply by LMSSonos** [reply]
@maxdd thanks, issue seems now finally solved, I deleted the app & cache from android, started fresh install. Backups are uploaded as expected and don't show twice. 

Personally I have the idea, it had something to do because the android migration process to the new phone moved over the settings and so probably in immich server it looked like the backup was done from the old phone and things got mixed up. but I don't know enough about the process in immich, so that's pure speculation ;-)

**Reply by maxdd** [reply]
I'm happy you found a solution although for me it happened with a single device. Irrespective of the failure cause it would be interesting to understand why the phone was not able to categorize two images the same.
